### DISCUSSION TITLE
how to  do Step4,  when i only have  multiple independent mrc files?

### DISCUSSION THREAD

**Discussion by yahulai2014** [mxhulab/cryopros#9] [General]
To convert multiple independent MRC files into MRC stack files???       and how to do it/?    '--data_path T00_HA_130K-Equalized-Particle-Stack**.mrcs** \'

**Comment by zhanghui186** [comment]
To create a particle stack, you can use the relion_stack_create command. Here are the specific instructions:
“relion_stack_create --i particles.star --o filename”
In this command, particles.star is the STAR file that contains all the particles. If you have performed preprocessing in cryoSPARC, you can use the "pyem" tool to convert the CS file to a STAR file before running the command.
